lms/djangoapps/teams/static/teams/js/spec/views/team_card_spec.js

```diff
@@ -47,17 +60,24 @@ define(['jquery',
 
             it('can render itself', function() {
                 expect(view.$el).toHaveClass('list-card');
-                expect(view.$el.find('.card-title').text()).toContain('Test Team');
-                expect(view.$el.find('.card-description').text()).toContain('A team for testing');
+                expect(view.$el.find('.card-title').text()).toContain(teamName);
+                expect(view.$el.find('.card-description').text()).toContain(description);
+                expect(view.$el.find('.card-type').text()).toContain(teamsetName);
                 expect(view.$el.find('.team-activity abbr').attr('title')).toContain('August 21st 2015');
                 expect(view.$el.find('.team-activity').text()).toContain('Last activity');
                 expect(view.$el.find('.card-meta').text()).toContain('0 / 100 Members');
                 expect(view.$el.find('.team-location').text()).toContain('United States of America');
                 expect(view.$el.find('.team-language').text()).toContain('English');
             });
 
+            it('does not show teamset name is showTeamset is false', function() {
```

Comment:
Why is the display of teamset name conditional?

Reply:
no reason to show teamset name if we're browsing teams for a specific topic

Reply:
that's the use case from Sapana.  Doesn't make sense to show teamset name on a page where that teamset is the heading of the page.

Reply:
^ To @jansenk's point, it may make sense to better explain the logic here.